tools/proofers/codepoint_validator.js

```diff
@@ -58,9 +62,67 @@ $(function () {
     });
 
     $("#cc-remove").click(function () {
-        let textarea = $("#text_data");
-        textarea.val(textarea.val().replace(pattern , ""));
+        textArea.value = textArea.value.replace(badPattern , "");
         $("#checker").hide();
         $("#proofdiv").show();
     });
+
+    var above = {
```

Comment:
We might also consider adding the following too, which would extend the current set of the DP diacritical markup:
```
        "*": "\u030A", // ring
        "\\": "\u0338", // stoke or bar
        "(": "\u0311", // inverted breve
```